xarray/tests/test_dataarray.py

```diff
@@ -2519,6 +2519,69 @@ def test_combine_first(self):
                              [('x', ['a', 'b', 'd']), ('y', [-1, 0])])
         self.assertDataArrayEqual(actual, expected)
 
+    def test_sortby(self):
```

Comment:
I would just add a single very basic test to verify that it works, and add a comment noting that more advanced functionality is tested in `test_dataset.py`.